src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -0,0 +1,238 @@
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import useAsync from 'react-use/esm/useAsync';
+
+import {Checkbox, TextInput} from '../../forms/Inputs';
+import Select from '../../forms/Select';
+import {DATATYPES_CHOICES} from './constants';
+import DeleteIcon from '../../DeleteIcon';
+import {PluginsContext} from '../Context';
+import {get} from '../../../../utils/fetch';
+import FAIcon from '../../FAIcon';
+import {ChangelistTableWrapper, HeadColumn} from '../../tables';
+
+const SensitiveData = ({isSensitive}) => {
+  const intl = useIntl();
+  const sensitiveTitle = intl.formatMessage({
+    description: 'Is sensitive icon title',
+    defaultMessage: 'Is sensitive',
+  });
+  const notSensitiveTitle = intl.formatMessage({
+    description: 'Is not sensitive icon title',
+    defaultMessage: 'Is not sensitive',
+  });
+
+  return isSensitive ? (
+    <FAIcon icon="exclamation-circle" title={sensitiveTitle} />
+  ) : (
+    <FAIcon icon="check-circle" title={notSensitiveTitle} />
```

Comment:
Been trying out some different icons, this is `.fa-regular .fa-circle-xmark`: 
![image](https://user-images.githubusercontent.com/5518550/169819244-791db235-99fa-4d67-8d5a-d0445e4d6f7e.png)

Other icons that might work: `.fa-ban`, `.fa-minus`.